test/ansible/config/manager/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/asmacdo/controller
+  newTag: v0.0.1
```

Comment:
Are these lines necessary? It looks like the default scaffolded file is usually just the `resources` field.